src/MadeTech/CacheWarming/Gateway/FileGetContentsUrlRetriever.php

```diff
@@ -7,6 +7,6 @@ class FileGetContentsUrlRetriever implements UrlRetriever
 {
     public function get($url)
     {
-        return file_get_contents($url);
+        return @file_get_contents($url);
```

Comment:
If we want to handle these errors, we should probably use something a bit more robust than file_get_contents. Probably Guzzle library. I'm not sure we need to go to that complexity, as in I'm not sure these errors are a usecase we care about